webapp/javascript/models/app.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from 'zod';
+
+export const appModel = z.object({
```

Comment:
@eh-am should we use uppercase in zod entities?

Reply:
like `appModel  => AppModel `

Reply:
or `AppSchema`

Reply:
I made names here like in this [users.ts](https://github.com/pyroscope-io/pyroscope/blob/525ba0c693080a2e0a48469a91d6c70ece1d1ab3/webapp/javascript/models/users.ts#L19) file